src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    protected final Randomizer randomizer;
+
+    public ColorSupplier() {
+        this.randomizer = new Randomizer();
+    }
```

Comment:
why `protected`?
```suggestion
    private final Randomizer randomizer = new Randomizer();
```

Reply:
cuz i don't wanna create a new Randomize object in class FigureSupplier/ getRandomFigure method.
but ok, i changed it